framework/wazuh/rbac/default/policies.yaml

```diff
@@ -8,6 +8,7 @@ default_policies:
         actions:
           - agent:create
           - group:create
+          - agent:uninstall
```

Comment:
Similar doubt related to the one above.

Reply:
Same explanation as https://github.com/wazuh/wazuh/pull/24621#discussion_r1684427336.